Mathlib/CategoryTheory/Limits/Shapes/Pullbacks.lean

```diff
@@ -2746,6 +2753,27 @@ def baseChange [HasPullbacks C] {X Y : C} (f : X ⟶ Y) : Over Y ⥤ Over X wher
     apply Over.OverMorphism.ext; apply pullback.hom_ext
     · dsimp; simp
     · dsimp; simp
-#align category_theory.limits.base_change CategoryTheory.Limits.baseChange
-
-end CategoryTheory.Limits
+#align category_theory.limits.base_change CategoryTheory.Over.baseChange
+
+/-- The adjunction `Over.map ⊣ baseChange` -/
+
```

Comment:
We usually do not put empty lines between the docstring comment and the declaration:
```suggestion
```